shell/src/main/java/alluxio/cli/bundler/CollectInfo.java

```diff
@@ -60,6 +61,17 @@
  */
 public class CollectInfo extends AbstractShell {
   private static final Logger LOG = LoggerFactory.getLogger(CollectInfo.class);
+  private static final String USAGE =
+          "USAGE: collectInfo [--max-threads <threadNum>] [--local]\n\n"
+                  + "collectInfo runs a set of sub-commands which collect information"
+                  + "about your Alluxio cluster. In the end of the run, "
+                  + "the collected information will be written to files "
```

Comment:
Usage should also reiterate credential risks. People don't read docs, but they generally take a look at the usage functionality.

Reply:
Marked as resolved as we move the code change part to #11037